frontend/src/hooks/useTrafficData.ts

```diff
@@ -0,0 +1,238 @@
+import { useState } from 'react';
+import type { IInstanceTrafficMetricsResponse } from './api/getters/useInstanceTrafficMetrics/useInstanceTrafficMetrics';
+import type { ChartDataset } from 'chart.js';
+
+const trafficDataUnitCost = 5;
+const trafficDataUnitSize = 1_000_000;
```

Comment:
I don't think we're super strict with this, but we generally identify constants like these with UPPER_SNAKE_CASE.